src/coreComponents/constitutive/fluid/PVTFunctions/BrineEnthalpy.cpp

```diff
@@ -0,0 +1,224 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file BrineEnthalpy.cpp
+ */
+
+#include "constitutive/fluid/PVTFunctions/BrineEnthalpy.hpp"
+
+#include "functions/FunctionManager.hpp"
+#include "constitutive/fluid/PVTFunctions/SpanWagnerCO2Density.hpp"
+#include "constitutive/fluid/PVTFunctions/CO2Enthalpy.hpp"
+
+
+namespace geosx
+{
+
+using namespace stringutilities;
+
+namespace constitutive
+{
+
+namespace PVTProps
+{
+
+namespace
+{
+
+real64 MichaelidesBrineEnthalpy( real64 const & T,
+                                 real64 const & m )
+{
+
+  static real64 a[4][3] = {
+    {-9633.6, -4080.0, 286.49},
+    {166.58, 68.577, -4.6856},
+    {-0.90963, -0.36524, 0.0249667},
+    {1.7965e-3, 7.1924e-4, -4.9e-5}
+  };
+
+  real64 x1, x2, h1, h2, dh;
+
+  x1 = 1000.0 / (1000.0 + 58.44 * m);
+
+  x2 = 58.44 * m  / (1000.0 + 58.44 * m);
+
+
+
+  dh = 0.0;
+
+  for( localIndex i = 0; i < 4; ++i )
+    for( localIndex j = 0; j < 3; ++j )
+    {
+
+      dh += a[i][j] * pow( T, real64( i )) * pow( m, real64( j ));
+
+    }
```

Comment:
It is just a small detail, but in other places in the code, we started replacing `localIndex` with `integer` for quantities that do not scale with the local mesh size (like the fluid phases, the fluid components, or here) to keep `localIndex` for quantities that really scale with the local mesh size (like the pressure field, etc). But no need to change this in your PR, it is good as is.

Reply:
Ok, thanks for explaining, understood!